felix/fv/Dockerfile.test.amd64

```diff
@@ -18,20 +18,18 @@
 # userspace environment as it will be deployed in calico/node.
 FROM ubuntu:focal as wgtool
 
-RUN apt-get update && \
```

Comment:
I'm not sure what happened here - this is failing on master for me as well. It seems like the second stage of the build doesn't have access to apt-get. 

TBH, I'm not sure we need this image at all any more - `calico/felix` isn't a released artifact so we could probably just combine this and calico/felix into one?